runtime/doc/luvref.txt

```diff
@@ -1,36 +1,24 @@
-*luvref.txt*               Nvim
-
-
-                    LUV REFERENCE MANUAL
-
-                                                                        *luvref*
-This file documents the Lua bindings for the LibUV library which is used for
-Nvim's event-loop and is accessible from Lua via |vim.uv| (e.g., |uv.version()|
-is exposed as `vim.uv.version()`).
-
-For information about this manual, see |luv-credits|.
-
-For further examples, see https://github.com/luvit/luv/tree/master/examples.
+*luvref.txt*                 For                 Last change: 2023 November 24
 
 ==============================================================================
-INTRODUCTION                                                  *luv* *luv-intro* *uv*
+1. LibUV in Lua                                          *luvref-libuv-in-lua*
```

Comment:
Is there any way of preserving the current tags? If not, that would be kind of a deal-breaker...

Reply:
I don't think so, I think the latest change is as close as we'll get at the moment.

Reply:
Then I'm afraid that's a -1 from me...

Reply:
Yeah kinda agree.